scala-package/init/src/main/scala/org/apache/mxnet/init/Base.scala

```diff
@@ -37,7 +37,10 @@ object Base {
 
   @throws(classOf[UnsatisfiedLinkError])
   private def tryLoadInitLibrary(): Unit = {
-    val baseDir = System.getProperty("user.dir") + "/init-native"
+    var baseDir = System.getProperty("user.dir") + "/init-native"
+    if (System.getenv().containsKey("MXNET_BASEDIR")) {
+      baseDir = sys.env("MXNET_BASEDIR") + "/scala-package/init-native"
```

Comment:
just FYI, I updated this line from 
`baseDir = sys.env("MXNET_BASEDIR")`
to 
`baseDir = sys.env("MXNET_BASEDIR") + "/scala-package/init-native"`